check_mandatory_fields.sh

```diff
@@ -12,42 +12,21 @@ set -e
 
 FIELDS=('displayName' 'description' 'tags' 'icon' 'globalMemoryLimit')
```

Comment:
This array var is different from the nearly-identical approach in the plugin registry. Could we use the same variables in both repos?

Reply:
I'd keep them separate mostly since their purposes are different and there's no requirement to sync metadata fields between plugins and devfiles. I'm mostly viewing these are loop variables, but I'll look at the registry as well and see it makes sense.